modules/tensor_mechanics/src/actions/TensorMechanicsAxisymmetricRZAction.C

```diff
@@ -35,17 +36,15 @@ TensorMechanicsAxisymmetricRZAction::act()
 {
   std::vector<NonlinearVariableName> displacements = getParam<std::vector<NonlinearVariableName> > ("displacements");
   std::vector<VariableName> coupled_displacements;
-  Real dim = displacements.size();
+  unsigned int dim = displacements.size();
 
   //Error checking:  Can only take two displacement variables in AxisymmetricRZ
   mooseAssert(dim == 2, "Expected two displacement variables but recieved " << dim);
 
   for (unsigned int i = 0; i < dim; ++i)
-  {
     coupled_displacements.push_back(displacements[i]);
-  }
 
-  // Retain this code 'as is' because StressDivergenceTensors inherits from Kernel.C
+  // Retain this code 'as is' because StressDivergenceAxisymmetricRZTensors inherits from StressDivergenceTensors
   std::vector<std::vector<AuxVariableName> > save_in;
   save_in.resize(dim);
```

Comment:
Although this is not code written in this PR, it has the same unecessary two line initialization pattern that @dschwen pointed out below.